src/main/java/io/vlingo/lattice/exchange/streams/ExchangeStreamPublisher.java

```diff
@@ -15,8 +15,9 @@
 
 /**
  * This class receives messages from an {@link Exchange} and streams them using {@link StreamPublisher} capabilities.
- * This class requires careful {@link PublisherConfiguration}, especially {@link Streams.OverflowPolicy} since inappropriate overflow policy
- * may fill up {@link ExchangeStreamSource} source.
+ * This class requires careful {@link PublisherConfiguration}, especially {@link Streams.OverflowPolicy} in conjunction with {@link ExchangeStreamSource} source.
+ * Low level values for {@link PublisherConfiguration#bufferSize} reduces memory usage (heap) while overflow drop rate may increase.
+ * On the other hand, high level values for {@link PublisherConfiguration#bufferSize} decrease overflow drop rate while increasing memory usage (heap).
```

Comment:
*from:* High level values *to:* High numeric values

Reply:
Updated!